elastic_monte_carlo_estimate_of_pi.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Monte Carlo Estimate of $\\pi$\n",
+    "\n",
+    "<img src=\"http://dask.readthedocs.io/en/latest/_images/dask_horizontal.svg\" \n",
+    "     width=\"50%\" \n",
+    "     align=top\n",
+    "     alt=\"Dask logo\">\n",
+    "<img src=\"https://upload.wikimedia.org/wikipedia/commons/b/ba/Monte-Carlo01.gif\" \n",
+    "     width=\"30%\" \n",
+    "     align=top\n",
+    "     alt=\"PI monte-carlo estimate\">\n",
+    "     \n",
+    "Using [Dask's adaptivity](http://docs.dask.org/en/latest/setup/adaptive.html), we'll show that it is possible to scale the available resources to meet almost identical wall times irrespective of the acutal work load:\n",
+    "\n",
```

Comment:
typo in actual.

You should maybe introduce also the Montecarlo estimate of py mechanism?